biz.aQute.bndlib/src/aQute/bnd/apiguardian/APIGuardianAnnotations.java

```diff
@@ -0,0 +1,66 @@
+package aQute.bnd.apiguardian;
+
+import java.util.Map.Entry;
+
+import aQute.bnd.header.Attrs;
+import aQute.bnd.header.OSGiHeader;
+import aQute.bnd.header.Parameters;
+import aQute.bnd.osgi.Analyzer;
+import aQute.bnd.osgi.Clazz;
+import aQute.bnd.osgi.Constants;
+import aQute.bnd.osgi.Instruction;
+import aQute.bnd.osgi.Instructions;
+import aQute.bnd.service.AnalyzerPlugin;
+
+public class APIGuardianAnnotations implements AnalyzerPlugin {
+
+	private final static String API_ANNOTATION = "org/apiguardian/api/API";
+	private final static String	INTERNAL_STATUS	= "INTERNAL";
+	private final static String	STATUS_PROPERTY	= "status";
+
+	@Override
+	public boolean analyzeJar(Analyzer analyzer) throws Exception {
+		Parameters header = OSGiHeader.parseHeader(analyzer.getProperty(Constants.EXPORT_APIGUARDIAN, "*"));
+		if (header.isEmpty())
+			return false;
+
+		Parameters exportPachages = analyzer.getExportPackage();
```

Comment:
I think you should be getting `getExportContents()` which means you will need to add the corresponding `setExportContents` which is missing from Domain.

Reply:
got it